biomaj/workflow.py

```diff
@@ -1674,7 +1674,7 @@ def wf_uncompress(self):
                 elif origFile.endswith('.zip'):
                     is_archive = True
                 else:
-                    raise Exception("Workflow:wf_uncompress: Unknown file extension in '" + str(archivefile) + "'")
+                    logging.warn("Workflow:wf_uncompress: file extension in '" + str(origFile) + "' is not treated as archive")
```

Comment:
not a warning... only for debug as done for each file
if you download lots of non-archive, you'll get hundreds of warning

Reply:
I removed it